lib/services/auth/firebase_auth_service.dart

```diff
@@ -21,9 +24,44 @@ class FirebaseAuthService implements AuthService {
 
   @override
   Future<UserData> signInWithEmail(String email, String password) async {
-    final authResult = await _firebaseAuth.signInWithEmailAndPassword(
-        email: email, password: password);
-    return authResult.user.toData();
+    try {
+      final result = await _firebaseAuth.signInWithEmailAndPassword(
+          email: email, password: password);
+      return result.user.toData();
+    } on PlatformException catch (error) {
+      String customErrorMessage;
+      switch (error.code) {
```

Comment:
Oh right, this is the error codes you mentioned.  I ran the app on an iOS simulator and created an account - there was no exception or error message displayed, it just stayed at the 'thinking' animation.  

I tried checking the db but I don't have access (which is fine with me just giving you the rundown of what I did) 

<img width="551" alt="Screen Shot 2020-08-12 at 1 11 42 am" src="https://user-images.githubusercontent.com/1059276/89914881-04343b80-dc39-11ea-8100-b28481eef2f2.png">

Reply:
Sorry, I see this is just for the sign in, not the create account.  I tried signing in and got the expected alert about no such user found.

Reply:
thank you, sorry ill be more specific next time :)